services/QuillLMS/e2e/main.spec.ts

```diff
@@ -1,28 +1,43 @@
 import { test, expect } from '@playwright/test';
 
+function filterAndLogRequest(request, regex) {
```

Comment:
Since we don't have access to the console `Network` tab in a headless CCI environment, these helper functions provide a stand-in.